lib/services/platform-service.ts

```diff
@@ -302,6 +309,16 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 
 		if (!changesInfo || changesInfo.modulesChanged) {
 			await this.copyTnsModules(platform, projectData);
+		} else if (appFilesUpdaterOptions.bundle) {
+			let dependencies = this.$nodeModulesDependenciesBuilder.getProductionDependencies(projectData.projectDir);
+			for (let dependencyKey in dependencies) {
+				const dependency = dependencies[dependencyKey];
+				let isPlugin = !!dependency.nativescript;
+				if (isPlugin) {
+					let pluginData = this.$pluginsService.convertToPluginData(dependency, projectData.projectDir);
+					await this.$pluginsService.preparePluginNativeCode(pluginData, platform, projectData);
```

Comment:
instead of exposing `preparePluginNativeCode` and `convertToPluginData` can't we just call `this.$pluginsService.prepare...` ?